src/ert/parsing/error_info.py

```diff
@@ -21,9 +21,11 @@ class ErrorInfo:
     originates_from: Optional[MaybeWithContext] = None
 
     @classmethod
-    def _take(cls, context: MaybeWithContext, attr: str):
+    def _take(cls, context: Union[MaybeWithContext, List[MaybeWithContext]], attr: str):
         if isinstance(context, FileContextToken):
             return context
+        elif isinstance(context, List):
+            return cls._take(context[0], attr)
```

Comment:
Are all items in this list the same? This function calls recursively, and it looks like you exit by just looking at the first item here.